apps/expo-go/android/expoview/src/main/java/host/exp/exponent/Constants.java

```diff
@@ -29,39 +29,15 @@ public static class ExpoViewAppConstants {
   private static final String TAG = Constants.class.getSimpleName();
 
   public static String VERSION_NAME = null;
-  public static String ABI_VERSIONS;
-  public static String SDK_VERSIONS;
-  public static List<String> SDK_VERSIONS_LIST;
+  public static String SDK_VERSION = ExponentBuildConstants.TEMPORARY_SDK_VERSION;
   public static final String TEMPORARY_SDK_VERSION = ExponentBuildConstants.TEMPORARY_SDK_VERSION;
   public static final String EMBEDDED_KERNEL_PATH = "assets://kernel.android.bundle";
   public static boolean DISABLE_NUX = false;
   public static int ANDROID_VERSION_CODE;
   public static boolean FCM_ENABLED;
   public static SplashScreenImageResizeMode SPLASH_SCREEN_IMAGE_RESIZE_MODE;
 
-  public static void setSdkVersions(List<String> sdkVersions) {
-    ABI_VERSIONS = TextUtils.join(",", sdkVersions);
-
-    // NOTE: Currently public-facing SDK versions and internal ABI versions are the same, but
-    // eventually we may decouple them
-    SDK_VERSIONS = ABI_VERSIONS;
-    SDK_VERSIONS_LIST = sdkVersions;
-  }
-
   static {
-    List<String> abiVersions = new ArrayList<>();
-    // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
-    // WHEN_PREPARING_SHELL_REMOVE_FROM_HERE
-    // ADD ABI VERSIONS HERE DO NOT MODIFY
```

Comment:
@tsapeta not quite sure how the versioning scripts will work going forward, but based on grep this string is no longer present. Assuming we'll just be using `TEMPORARY_SDK_VERSION` until we rename/replace it.